common/src/main/java/com/sequenceiq/cloudbreak/logger/MDCContextFilter.java

```diff
@@ -43,18 +43,13 @@ public MDCContextFilter() {
 
     @Override
     protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws IOException, ServletException {
-        MDCBuilder.cleanupMdc();
-        HttpServletRequestWrapper wrapper = new RequestIdHeaderInjectingHttpRequestWrapper(request);
-        Builder builder = MdcContext.builder().requestId(wrapper.getHeader(REQUEST_ID_HEADER));
```

Comment:
Since this comes later in the chain, will this end up resetting the MDC and not setting up the new one with a requestId?

Reply:
Yes, that needs to be removed, correct.